templates/docker-compose/restore.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+. .env
+LAST_BACKUP_DIR="backup/$(ls -1t backup/ | head -n1)"
+LAST_BACKUP_FILE=$(ls -1 ${LAST_BACKUP_DIR} | grep 'app-.*gz')
```

Comment:
The new backups probably need some kind of version prefix. For example, the old `backup.sh` generated `backup/1666881527-2022-10-27` and that's going to cause some odd results if `restore.sh` picks it up

Reply:
Only looks for new `app*.tar.gz` format now